common/utils/dates.test.ts

```diff
@@ -214,3 +215,29 @@ describe('getDatesBetween', () => {
     ]);
   });
 });
+
+describe('countDaysBetween', () => {
+  test.each([
```

Comment:
A nice edge case to test would be with a different year and/or a different month?

Reply:
Good suggestion, added.